.github/workflows/test.yaml

```diff
@@ -138,6 +158,7 @@ jobs:
       - name: Run integration
         # Force one single concurrent test
         run: tox -e integration
+        continue-on-error: true  # so that other juju versions are tested
```

Comment:
Am I understanding correctly that `continue-on-error` being `true` here means that a failure won't prevent the other Juju versions from being run, but `continue-on-error=false` at the `integration` level means that a failure will still cause the `integration` job as a whole to be marked as failed?

Reply:
Yes, that's the intention anyway.